packages/lesswrong/components/ea-forum/AdvisorCard.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import LocationIcon from '@material-ui/icons/LocationOn'
+import { socialMediaIconPaths } from '../form-components/PrefixedInput';
+
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    maxWidth: 290
+  },
+  profileImage: {
+    width: 96,
+    height: 96,
+    background: theme.palette.grey[100],
+    'box-shadow': `3px 3px 1px ${theme.palette.boxShadowColor(.25)}`,
+    '-webkit-box-shadow': `0px 0px 2px 0px ${theme.palette.boxShadowColor(.25)}`,
+    '-moz-box-shadow': `3px 3px 1px ${theme.palette.boxShadowColor(.25)}`,
+    borderRadius: '50%',
+    marginBottom: 14,
+  },
+  name: {
+    fontSize: 16,
+    fontWeight: 700
+  },
+  role: {
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 12,
+    marginTop: 4
+  },
+  org: {
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 11,
+    marginTop: 4
+  },
+  locationRow: {
+    display: 'flex',
+    alignItems: 'baseline',
+    columnGap: 20,
+    color: theme.palette.grey[700],
+    fontSize: 12,
+    marginTop: 9
+  },
+  location: {
+    display: 'flex',
+    alignItems: 'center',
+    columnGap: 3,
+  },
+  locationIcon: {
+    fontSize: 14,
+  },
+  socialMediaIcons: {
+    display: 'flex',
+    columnGap: 6,
+  },
+  socialMediaIcon: {
+    flex: 'none',
+    height: 16,
+    fill: theme.palette.grey[600],
+  },
+  askMeAbout: {
+    color: theme.palette.grey[500],
+    fontSize: 10,
+    marginTop: 16
+  },
+  topics: {
+    fontFamily: theme.typography.fontFamily,
+    color: theme.palette.grey[700],
+    fontSize: 12,
+    lineHeight: '19px',
+    paddingInlineStart: '16px',
+    margin: 0
+  }
+})
+
+const AdvisorCard = ({user, classes}: {
+  user: {
```

Comment:
Nitpick: it might be nice to add some `type Advisor` that defines this. We can then just use that here, and we can strongly-type the array of advisors as an `Advisor[]`.